bedrock/base/templates/macros.html

```diff
@@ -583,7 +583,8 @@
             <li><a href="{{ url('firefox.sync') }}" data-id="android-sync"><div>{{ _('Sync') }}</div></a></li>
           </ul>
         </li>
-        <li>
+        <li class="fxos">
+          <span class="separator"></span>
```

Comment:
Oh I see, you're using <code>:before</code> and <code>:after</code> on the span itself... maybe this is necessary after all.